templates/DatasetReview/submission-form.html.twig

```diff
@@ -83,8 +83,11 @@
             </div>
 
             <div id="ds-submit" class="ds-metadata">
+                <div id="dropzone-uploader" class="file-dropzone-area">
```

Comment:
Move this to the filemanager component, and use the same ID.
Also disable previewTemplate.